<launch>
    <param name="node_type" type="str" value="topic" />
    <node name="ros_node" type="segmentation_node.py" pkg="segmentation" output="screen"/>
</launch>